### PR TITLE
CSP-928 AAN Apprentice Directory Message Apprentice Web

### PR DESCRIPTION
**Pull request by ManishAgarwalDfE**

### CODE REVIEW

File: src/SFA.DAS.ApprenticeAan.Web.UnitTests/Controllers/MemberProfileControllerTests.cs [hunk 1]
```diff
@@ -68,6 +74,7 @@ public class MemberProfileControllerTests
         MemberUserType memberUserType,
         [Frozen] Mock<IOuterApiClient> outerApiMock,
         GetMemberProfileResponse getMemberProfileResponse,
+        Mock<IValidator<SubmitConnectionCommand>> validator,
```
Comment (MarkFCain) [inline src/SFA.DAS.ApprenticeAan.Web.UnitTests/Controllers/MemberProfileControllerTests.cs:77]: rename to validatorMock, makes it easier to work out the test

File: src/SFA.DAS.ApprenticeAan.Web.UnitTests/Controllers/MemberProfileControllerTests.cs [hunk 2]
```diff
@@ -136,4 +143,146 @@ public void MemberProfileDetailMapping_ReturnsMemberProfileDetail(MemberUserType
...
+
+    [Test]
+    [MoqInlineAutoData(MemberUserType.Apprentice)]
+    [MoqInlineAutoData(MemberUserType.Employer)]
+    public async Task Post_InvalidCommand_ReturnsMemberProfileView(
+        MemberUserType userType,
+        SubmitConnectionCommand command,
+        [Frozen] Mock<IOuterApiClient> outerApiMock,
+        GetMemberProfileResponse getMemberProfileResponse,
+        Mock<IValidator<SubmitConnectionCommand>> validator,
```
Comment (MarkFCain) [inline src/SFA.DAS.ApprenticeAan.Web.UnitTests/Controllers/MemberProfileControllerTests.cs:155]: rename to validatorMock

File: src/SFA.DAS.ApprenticeAan.Web.UnitTests/Controllers/MemberProfileControllerTests.cs (hunk 1 above)
Comment (ManishAgarwalDfE) [reply src/SFA.DAS.ApprenticeAan.Web.UnitTests/Controllers/MemberProfileControllerTests.cs]: Done

File: src/SFA.DAS.ApprenticeAan.Web.UnitTests/Controllers/MemberProfileControllerTests.cs (hunk 2 above)
Comment (ManishAgarwalDfE) [reply src/SFA.DAS.ApprenticeAan.Web.UnitTests/Controllers/MemberProfileControllerTests.cs]: Done

**Review by MarkFCain — DISMISSED** [review]

**Review by MarkFCain — APPROVED** [review]

**Review by MarkFCain — APPROVED** [review]

**Pull request merged**